### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by galexandre**
Add some useful plateforms

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://houndci.com/ - Comments on github commits about code quality - free for open source
   * https://coveralls.io/ - Display test coverage reports - free for open source
   * https://scrutinizer-ci.com/ - Continuous inspection platform - free for open source
+  * https://codecov.io/ - Code coverage tool (SaaS) free for 1 private project and no restructions for publics repos
```
Comment (ripienaar) [inline README.md]: "restrictions"

File: README.md [hunk 2]
```diff
@@ -36,6 +37,8 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://travis-ci.org – free for public Github repositories.
   * http://wercker.com/ - free for public and private repositories
   * https://drone.io/ - CI platform that includes browser testing, free for Open Source
+  * https://jenkins-ci.org/ - CI software
+  * http://hudson-ci.org/ - CI software
```
Comment (ripienaar) [inline README.md]: hmm, think I am keen on hosted services rather than free self hosted stuff

**Comment by galexandre** [comment]
Ho sorry for the mistake. 

you put Sonar, so I think we can add Jenkins and Hudson

**Comment by ripienaar** [comment]
yeah good catch, I'll remove sonar

**Comment by galexandre** [comment]
;) 
Do you know some equivalents of Sonar ?

**Comment by galexandre** [comment]
I removed Hudson, Jenkins and Sonar and I corrected the mistake.

**Comment by ripienaar** [comment]
nice one :)

**Pull request merged**